src/sql/workbench/contrib/notebook/browser/cellViews/code.css

```diff
@@ -33,6 +33,7 @@ code-component .toolbar .carbon-taskbar {
 	position: sticky;
 	top: -16px;
 	margin-top: 5px;
+	height: 29px;
```

Comment:
29px is from the height of the icon https://github.com/microsoft/azuredatastudio/blob/1f27c3b942e5ecd464c30bf6bbc98b419499b884/src/sql/workbench/contrib/notebook/browser/cellViews/code.css#L40
and padding https://github.com/microsoft/azuredatastudio/blob/a180a6bc4719fa0292a03cfbd2faf9bb64404fa5/src/sql/base/browser/ui/taskbar/media/taskbar.css#L25